From b86361fa4e5e3b88a920f76cbb5434ef2b94a0c2 Mon Sep 17 00:00:00 2001
From: Jason Song <sxj@rock-chips.com>
Date: Thu, 29 Oct 2020 11:45:13 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Enable CONFIG_TOUCHSCREEN_FTS

Enable focaltech touch driver used on Rockchip RK3566 RKG11 2K TABLET.

Signed-off-by: Jason Song <sxj@rock-chips.com>
Change-Id: I4c25d9a31f697b49e36d3d653435013fe57f9c84
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 6475f8162852..f3501750e267 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -440,6 +440,7 @@ CONFIG_TOUCHSCREEN_GSL3676=y
 CONFIG_TOUCHSCREEN_GT9XX=y
 CONFIG_TOUCHSCREEN_HYN_CST2XX=y
 CONFIG_TOUCHSCREEN_GT1X=y
+CONFIG_TOUCHSCREEN_FTS=y
 CONFIG_ROCKCHIP_REMOTECTL=y
 CONFIG_ROCKCHIP_REMOTECTL_PWM=y
 CONFIG_SENSOR_DEVICE=y
-- 
2.35.3

